testing/trino-product-tests/src/main/java/io/trino/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -1280,6 +1280,24 @@ public void testDeletePartitionedTable()
         });
     }
 
+    @Test(groups = HIVE_TRANSACTIONAL, timeOut = TEST_TIMEOUT)
+    public void testUpdatePartitionedTable()
```

Comment:
I don't feel like this test is needed over `testAcidUpdatePartitioned`

if you keep it, please give it a name that can distinguish it.
currently the two have names saying the same ("updated partitioned table"), just with different words

Reply:
You're right, we can remove it